Dockerfile

```diff
@@ -3,17 +3,15 @@ FROM centos:7
 # Install dependencies
 RUN yum -y install epel-release  && \
     yum -y install bash wget unzip \
-           pexpect python-daemon  bubblewrap gcc \
-           bzip2  openssh openssh-clients python2-psutil\
-           python36 python36-devel python36-setuptools\
-           nginx supervisor && \
-           localedef -c -i en_US -f UTF-8 en_US.UTF-8
-RUN easy_install-3.6 -d /usr/lib/python3.6/site-packages pip && \
-    ln -s /usr/lib/python3.6/site-packages/pip3 /usr/local/bin/pip3
+    pexpect python-daemon  bubblewrap gcc \
+    bzip2  openssh openssh-clients python2-psutil\
+    python36 python36-devel python36-setuptools\
+    nginx supervisor && \
+    localedef -c -i en_US -f UTF-8 en_US.UTF-8
 
-RUN /usr/local/bin/pip3 install ansible cryptography docutils psutil PyYAML \
-                 pyOpenSSL flask flask-restful uwsgi netaddr notario && \
-    /usr/local/bin/pip3 install --no-cache-dir ansible-runner==1.3.2 && \
+RUN /usr/bin/python3 -m pip install ansible cryptography docutils psutil PyYAML \
+    pyOpenSSL flask flask-restful uwsgi netaddr notario && \
+    /usr/bin/python3 -m pip install --no-cache-dir ansible-runner==1.3.2 && \
```

Comment:
should we also take the opportunity to update this version for the upstream container? 1.3.2 is quite old, and when mixed with the latest parallel changes files end up in weird places within artifacts

Reply:
I have updated to the latest available 1.4.6 (March 26). I have a quick test playng with inventory and executing the test playbooks. It seems that it is working ok.
@mwperina @mnecas @machacekondra Are you ok with this change?